ftrackhq/ftrack-connect-pipeline
# feat: Backlog/unreal/project context/implement snapshot assets

Resolves 


## Changes

- Publish Dependencies Asset info as metadata of the current asset [Batch publisher]
- Create asset publisher [Publisher]
- Implement asset tabs to AM [Unreal Assets] [pipeline Assets]
- Create snapshot asset info on dependency batch publish
- Check modified dependencies [Publisher]
- Recursively import dependencies [Loader, Opener]
- Check asset loader (uasset) [Loader]

## Test

_Note: has to be tested with corresponding branch on pipeline, definition, qt and unreal_

- Load FBX > set material > publish asset
- Open asset in new project, material should be imported with asset.
- Open level task > load asset in assembler > check AM that all assets are listed.
